__tests__/app.test.js

```diff
@@ -50,6 +51,7 @@ describe('GET /api/articles/:article_id', () => {
                 body: 'I find this existence challenging',
                 created_at: '2020-07-09T20:11:00.000Z',
                 votes: 100,
+                comment_count: 11
```

Comment:
Usually, we would try to avoid editing our old tests and create a new one. Don't worry about changing it now, but just a tip!

Reply:
Thanks :)